Makefile

```diff
@@ -105,3 +105,12 @@ deploy: examples/admission-cert.pem
 		|sed "s/__admission_cert__/$(shell cat examples/admission-cert.pem|base64 -w0)/g" \
 		|sed "s/__admission_key__/$(shell cat examples/admission-key.pem|base64 -w0)/g" \
 		|kubectl apply -f -
+
+lint:
+	./hack/verify-type-revision-annotation-const.sh
```

Comment:
There is btw already a lint target on line 39

Reply:
:woman_facepalming: thanks, will fix.